explorer/src/providers/transactions/details.tsx

```diff
@@ -55,9 +116,16 @@ async function fetchDetails(
   let fetchStatus;
   let transaction;
   try {
-    transaction = await new Connection(url).getParsedConfirmedTransaction(
-      signature
-    );
+    const tx = await RetrieveSignature(signature, `${solarweave}-index`);
```

Comment:
Is this database specific to signatures? I would expect the name to be something like `${solarweave}-signatures`

Reply:
I'd like this to be in a nested try catch so that if for some reason the arweave api request fails, we still fall back to the RPC api

Reply:
The reason it's `index` is because the `index` database is able to retrieve blocks based on account keys and signatures. So it encapsulates multiple tags.

The `try` `catch` statement is essential though!